### PR TITLE
Tracking system

### PR DESCRIPTION
**Pull request by hazelwoodA**
Team Members (1-4 suggested): 2, Myself and my co-leader. (Spencer U06MJLY1GJU, Silvia U06M89VS2SD on Slack)

Is this your first PCB rodeo? (if yes, don't feel bad. you're all good): Yes.

What you are making? We will be making small keychains/'pings' for hikers that are able to be tracked by leaders in the hike on a separate device. 

Inspiration: Apple airtags and personal alarms.

Sketches (if they exist) (paper works): None currently

### CODE REVIEW

File: equipment/Tracking-System [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Tracking system
```
Comment (Luna5379) [inline equipment/Tracking-System:1]: (catchy name incoming)

**Comment by 24c02** [comment]
hi! can you tell me more about how you might go about implementing this?

**Comment by hazelwoodA** [comment]
@24c02 , hey! Our main aims for this project are a tracking system leaders can look at, with keychain-sized trackers for individuals in groups. These could use the network infrastructure that I heard being mentioned, but we'll have to look deeper into that as we start building the PCB. Another part of it is that users should have a button that will communicate back to a leader an SOS signal if necessary. The leader's tracking system would be a map of the area on maybe an OLED screen with a map of the area, and each node signaling to the leader's receiver. We'd still have to fine-tune this to figure out the exact equipment and get started with our PCB designs and what we'd use, budgeting for each, etc. I hope this answers your question?

**Comment by 24c02** [comment]
i guess we'll figure out the details later?

**Pull request merged**